spec/aruba/api/commands_spec.rb

```diff
@@ -6,8 +6,10 @@
   include_context 'uses aruba API'
 
   describe '#run_command' do
+    let(:cmd) { 'ruby -ne "puts $_"' }
```

Comment:
Would something lets `gets.chomp` work here. Via an input on stdin? Genuinely not sure, so ignore me if not useful.

Reply:
That almost works, but the process exits immediately afterwards. I can add a loop and trap an interrupt signal, but it's a fair bit more verbose. I've also found it doesn't _quite_ function in the same way that `cat` does. Thanks heaps for the suggestion, though. :)

Reply:
@xtrasimplicity I can't remember what the Windows-equivalent command was. `type`, maybe? I kind of like your change because it's Just Ruby, but it may slow down tests on JRuby. Let's see how this goes for now.

Reply:
Thanks, @mvz. `type` was almost perfect, from memory, but I don't think it handled `SIGINT` signals (or the windows equivalent) correctly. I'll keep it as is for the moment and see how it goes. :)